src/client/pages/VergunningDetail/ERVV.tsx

```diff
@@ -1,30 +1,37 @@
 import { ERVV as ERVVType } from '../../../server/services';
-import { defaultDateFormat } from '../../../universal/helpers';
+import { defaultDateTimeFormat } from '../../../universal/helpers';
 import InfoDetail, {
   InfoDetailGroup,
 } from '../../components/InfoDetail/InfoDetail';
 import { Location } from './Location';
+import { defaultDateFormat } from '../../../universal/helpers/date';
 
 export function ERVV({ vergunning }: { vergunning: ERVVType }) {
   return (
     <>
       <InfoDetail label="Kenmerk" value={vergunning?.identifier || '-'} />
-      <InfoDetail label="Zaaktype" value={vergunning.caseType || '-'} />
+      <InfoDetail label="Soort vergunning" value={vergunning.caseType || '-'} />
       <InfoDetail label="Omschrijving" value={vergunning?.description || '-'} />
       {!!vergunning.location && <Location location={vergunning.location} />}
       <InfoDetailGroup>
         <InfoDetail
           label="Vanaf"
           value={
-            vergunning?.dateStart
-              ? defaultDateFormat(vergunning.dateStart)
+            vergunning?.timeStart
```

Comment:
Betekent dit dat wij dateStart + dateEnd niet gebruiken?

Reply:
Ah ik zie nu in de update van het RD de clue. Het moet hier een combinatie zijn van dateStart en timeStart. Handig om dit in de Frontend te formatten dan denk ik.

Reply:
Dat kan, timeStart + timeEnd komt al als tijd uit de api. Dus je zou iets kunenn doen als:

```
let start = vergunning?.dateStart ? defaultDateFormat(vergunning.dateStart) : '';
if (vergunning.timeStart) {
  start += ` ${vergunning.timeStart}`;

}

value={start.trim()}
```